backend/src/api/database-migration.ts

```diff
@@ -632,6 +637,11 @@ class DatabaseMigration {
       queries.push(`INSERT INTO state(name, number, string) VALUES ('last_weekly_hashrates_indexing', 0, NULL)`);
     }
 
+    if (version < 55) {
```

Comment:
This did not execute. Wrong version number here?

Reply:
Yep, forgot this one while rebasing. Fixed in https://github.com/mempool/mempool/pull/3122/commits/e0c3c732d17d4c05831245f7f07c85def8a3d19e.